addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js

```diff
@@ -348,13 +348,23 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
       this.storeOptions.sourceGraph
     );
 
-    const validCosts = entries.filter(entry => parseInt(entry.object.value) > 0 );
-    if(!validCosts.length){
+    const inValidCosts = entries.filter(entry => isNaN(parseInt(entry.object.value)));
+    if(inValidCosts.length) {
```

Comment:
very small detail, but shouldn't it be `invalidCosts` instead of `inValidCosts` ? Made me confused for a sec :see_no_evil:

Reply:
Oh my camelCase mind bugged out there, yeah indeed invalid is one word :p